.travis.yml

```diff
@@ -6,15 +6,12 @@ cache:
 python:
   - '3.6'
   - '3.7'
+before_install:
+- pip install poetry
 install:
-- pip install -r dev-requirements.txt
-- pip install -e .
-- pip install coveralls==1.3.0
-- pip list
+- poetry install
 script:
 - travis_wait py.test tests --cov rasa_sdk -v
-after_success:
-- coveralls
```

Comment:
are we still reporting to coveralls though?

Reply:
seems like we don't (at least not for rasa-sdk): https://coveralls.io/github/RasaHQ